torch/_inductor/codegen/common.py

```diff
@@ -518,6 +518,11 @@ def _print_RoundDecimal(self, expr):
         assert isinstance(ndigits, sympy.Integer)
         return f"round({self._print(number)}, {ndigits})"
 
+    def doprint(self, expr, *, simplify: bool = True):
+        if simplify:
```

Comment:
Why is here better than before calling pexpr?

Reply:
There are a lot of pexpr calls so I figured I wanted to hit all of them